src/main/java/fileservice/FileServiceImpl.java

```diff
@@ -0,0 +1,62 @@
+package fileservice;
+
+import errors.ErrorWritingDataToFile;
+import errors.InvalidFileExtension;
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.util.ArrayList;
+import java.util.List;
+
+public class FileServiceImpl implements FileService {
+    @Override
+    public List<String> read(String file) {
+        if (!file.endsWith(".csv")) {
```

Comment:
if your file can work only with csv files, probably worth to name it like CsvFileServiceImpl